core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +60,30 @@ def map(exploration_model):
     def reduce(exp_id, list_of_exps):
         pass
 
+class ExpSummariesContributorsOneOffJob(jobs.BaseMapReduceJobManager):
```

Comment:
Nit: two lines above top-level class def.

Also, there should be an entry in job_registry for this?